meeting.js

```diff
@@ -0,0 +1,330 @@
+      microphoneButton.innerHTML = '<i class="fas fa-microphone icon green"></i>';
+      microphoneButton.style.padding = "14px 21px";
+      bnrButton.classList.remove('selected');
+      vbgButton.classList.remove('selected');
+      createdMeeting=null;
+      localStream = null;
+      isMuted = true;
+      isVideoStarted = true;
+      vbgEffect = false;
+      
+    }
+  } catch (error) {
+    console.error('Error leaving meeting:', error);
+    $('#joinMeetingModal').modal('hide');
+
+  }
+}
+
+// 12) Start and Stop Button using toggleVideo Function.
+videoButton.addEventListener('click', toggleVideo);
+async function toggleVideo() {
+  if (!localStream || !localStream.camera || !localStream.camera.outputStream) {
+    console.error('No local video stream available.');
+    return;
+  }
+  // Stop Video
+  if (isVideoStarted) {
+    await createdMeeting.unpublishStreams([localStream.camera]);
+    localVideo.srcObject = null;
+    document.getElementById("localvideoimage").style.display = "block";
+    videoButton.innerHTML = '<i class="fas fa-video-slash icon red"></i>';
+    isVideoStarted = false;
+    vbgButton.classList.remove('selected');
+    if (vbgEffect && vbgEffect.isEnabled) {
+      await vbgEffect.disable();
+    }
+  } 
+  // Start Video
+  else {
+    const cameraStream = await webex.meetings.mediaHelpers.createCameraStream({ width: 640, height: 480 });
+    localStream.camera = cameraStream;
+    localVideo.srcObject = cameraStream.outputStream;
+    await createdMeeting.publishStreams({ camera: localStream.camera });
+    videoButton.innerHTML = '<i class="fas fa-video icon green"></i>';
+    document.getElementById("localvideoimage").style.display = "none";
+    isVideoStarted = true;
+  }
+}
+
+// 13) Mute and UnMute Button using toggleMicrophone Function
+microphoneButton.addEventListener('click', toggleMicrophone);
+async function toggleMicrophone() {
+  if (!localStream) {
+    console.error('No local stream available.');
+    return;
+  }
+  // Unmute Local Audio
+  if (!isMuted) {
+    try {
+      const microphoneStream = await webex.meetings.mediaHelpers.createMicrophoneStream({
+        echoCancellation: true,
+        noiseSuppression: true,
+      });
+
+      localStream.microphone = microphoneStream;
+      await createdMeeting.publishStreams({ microphone: localStream.microphone });
+      microphoneButton.innerHTML = '<i class="fas fa-microphone icon green"></i>';
+      microphoneButton.style.padding = "14px 21px";
+      isMuted = true;
+    } catch (error) {
+      console.error('Error creating microphone stream:', error);
+    }
+  } 
+  // Mute Local Audio
+  else {
+    if (localStream.microphone) {
+      await createdMeeting.unpublishStreams([localStream.microphone]);
+      localStream.microphone = null;
+      bnrButton.classList.remove('selected');
+    }
+    microphoneButton.innerHTML = '<i class="fas fa-microphone-slash icon red"></i>';
+    microphoneButton.style.padding = "14px 17px";

```

Comment:
Same comment here. Applicable wherever we have made inline CSS change

Reply:
No Kesava when we use different icon for UnMute The padding was different due to icons size.I have tried by changing the icon size but it did not work so i had use the padding.

Reply:
The font-awesome icons should have an equal sized icon set. Even if we are adjusting padding, it should be done from CSS and not from Javascript.

Better to write CSS based on the icon class names and adding padding there so it adjusts automatically

Reply:
Like All The Styles Were perfect but The Icon left padding didn't work So I had added the js .So If needed I can add Seperate CSS And add it as an Class list